manifests/c/cryptodarth/dockerbeam/0.1.0.0/cryptodarth.dockerbeam.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: cryptodarth.dockerbeam
+PackageVersion: 0.1.0.0
+InstallerType: portable
+Commands:
+- dockerbeam
+Installers:
+- Architecture: x86
```

```suggestion
Dependencies: 
  PackageDependencies: 
    - Microsoft.VCRedist.2015+.x64
Installers:
- Architecture: x64
```
This package has a runtime dependency on VCRedist. I tested with VCRedist x86, and it still did not work, which means that despite being a 32-bit exe, it requires an x64 OS to satisfy the VCRedist dependency

---

Thank you for the suggestions.
I have commited to my manifest , should i open another PR ?